akka-remote/src/main/scala/akka/remote/artery/OutboundEnvelope.scala

```diff
@@ -52,16 +56,30 @@ private[akka] final class ReusableOutboundEnvelope extends OutboundEnvelope {
   private var _recipient: OptionVal[RemoteActorRef] = OptionVal.None
   private var _message: AnyRef = null
   private var _sender: OptionVal[ActorRef] = OptionVal.None
+  private val _metadata: MetadataMap[AnyRef] = MetadataMap()
```

Comment:
Removed this field from this PR from `ReusableOutboundEnvelope` cc @bantonsson you'll have to add it yourself and populate from `send`